github-event-service/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import * as express from 'express';
+
+import {handlePushEvent} from './push';
+
+/** 
+ * Handle events from github webooks.
+ * 
+ * The expected events from Github are `pull_request` and `push`.
+ */
+export const githubEvents = (req: express.Request, res: express.Response) => {
+  res.setHeader('Content-Type', 'application/json');
+  // Github descibes what the type of the webhook event trigger was, and thus
+  // what the structure of the request payload will be via the X-Github-Event
+  // payload .
```

Comment:
Would be probably good to add `https://developer.github.com/webhooks/#payloads` here to the comment.